File: News_Service_JDBC/src/main/java/com/dunice/projectNews/services/AuthService.java
package com.dunice.projectNews.services;

import com.dunice.projectNews.dto.request.AuthRequest;
import com.dunice.projectNews.dto.request.RegisterUserRequest;
import com.dunice.projectNews.dto.response.LoginUserResponse;

public interface AuthService {
    LoginUserResponse registration(RegisterUserRequest registerUserDtoRequest);

    LoginUserResponse login(AuthRequest authRequest);
}

File: News_Service_Kotlin/src/main/kotlin/com/dunice/advancedServiceKotlin/services/AuthService.kt
package com.dunice.advancedServiceKotlin.services

import com.dunice.advancedServiceKotlin.dto.request.AuthRequest
import com.dunice.advancedServiceKotlin.dto.request.RegisterUserRequest
import com.dunice.advancedServiceKotlin.dto.response.LoginUserResponse

interface AuthService {
    fun registration(registerUserDtoRequest: RegisterUserRequest): LoginUserResponse

    fun login(authRequest: AuthRequest): LoginUserResponse
}
